app/components/Editor/Editor.tsx

```diff
@@ -143,6 +145,16 @@ export function Editor({ protoInfo, initialRequest, onRequestChange }: EditorPro
     metadata: (initialRequest && initialRequest.metadata) || getMetadata() || INITIAL_STATE.metadata,
   }, undefined);
 
+  const commands: Command[] = [
+    {
+      name: 'Request',
+      bindKey: { win: 'Ctrl+Enter', mac: 'Command+Enter' },
+      exec: () => {
+        makeRequest({ protoInfo, state, dispatch });
```

Comment:
This won't work as the state will be a reference to the old state. Tested, and fixes in https://github.com/uw-labs/bloomrpc/pull/215